build/repo.props

```diff
@@ -166,6 +166,7 @@
                           $(RepositoryRoot)src\MusicStore\**\*.*proj;
                           $(RepositoryRoot)src\SignalR\**\*.csproj;
                           $(RepositoryRoot)src\Components\**\*.csproj;
+                          $(RepositoryRoot)src\Analyzers\**\*.csproj;
```

Comment:
@natemcmaster does order matter here? Right before templates seemed like the right place if it does.

Reply:
Order does not matter.